ci/setup_cmake.sh

```diff
@@ -4,13 +4,19 @@ set -e
 
 export DEBIAN_FRONTEND=noninteractive
 apt-get update
-apt-get install --no-install-recommends --no-install-suggests -y \
+
+set +e
+echo		\
                 cmake \
                 libbenchmark-dev \
                 libgtest-dev \
                 zlib1g-dev \
                 sudo \
-                libcurl4-openssl-dev
+                libcurl4-openssl-dev \
+                nlohmann-json-dev \
```

Comment:
Main reason why I need to ignore the errors / skip missing, is because the package name / version is different between distros. I'm trying to keep this setup script universal to work for any recent distro (Ubuntu 16.xx, 18.xx, 20.xx).. Basically, we "install whatever available" json.hpp . Ideally we may want to reconsider, and either use PPA to install latest, OR add specific version we need as git submodule.